.make/test.mk

```diff
@@ -542,3 +573,13 @@ CLEAN_TARGETS += clean-coverage-remote
 ## Removes remote test coverage file
 clean-coverage-remote:
 	-@rm -f $(COV_PATH_REMOTE)
+
+CLEAN_TARGETS += clean-e2e
+.PHONY: clean-e2e
+clean-e2e:
```

Comment:
Please add a description beginning with `##` above this line so that `make help` can pick it up.

Reply:
Done

Reply:
Done